src/PlusDataCollection/vtkPlusDeviceFactory.cxx

```diff
@@ -312,6 +316,10 @@ vtkPlusDeviceFactory::vtkPlusDeviceFactory()
   RegisterDevice("OpenCVVideo", "vtkPlusOpenCVCaptureVideoSource", (PointerToDevice)&vtkPlusOpenCVCaptureVideoSource::New);
 #endif
 
+#ifdef PLUS_USE_OPENHAPTICS
+  RegisterDevice("OpenHapticsDevice", "vtkPlusOpenHapticsDevice", (PointerToDevice)&vtkPlusOpenHapticsDevice::New);

```

Comment:
Would you remove the Device suffix from the identifier (ie: just OpenHaptics)? Also matching identifier in .dox file